src/AISmart.GAgent.Core/IEventDispatcher.cs

```diff
@@ -5,4 +5,5 @@ namespace AISmart.GAgent.Core;
 public interface IEventDispatcher
 {
     Task PublishAsync(StateBase state, string id);
+    Task PublishGEventAsync(GEventBase eventBase, string id);
```

Comment:
Both publish has the same domain usage. They should both be named PublishAsync instead.